crates/ruff_cli/src/commands/mod.rs

```diff
@@ -0,0 +1,17 @@
+pub use add_noqa::add_noqa;
+pub use clean::clean;
+pub use linter::linter;
+pub use rule::rule;
+pub use run::run;
+pub use run_stdin::run_stdin;
+pub use show_files::show_files;
+pub use show_settings::show_settings;
+
+mod add_noqa;
+mod clean;
+mod linter;
+mod rule;
+mod run;
+mod run_stdin;
+mod show_files;
+mod show_settings;
```

Comment:
Just a note that in general I prefer to make the modules public and use e.g. `commands::rule::rule` in `main.rs`.

Reply:
I'm not disagreeing, but why do you prefer that style? (Just trying to learn. I wasn't really sure myself what was preferable.)

(The one thing I _dislike_ is when there are multiple ways to import the same member due to a re-export, e.g., you can import `RuleSelector` via `use crate::rule_selector::RuleSelector;` or `use crate::RuleSelector;` in `ruff`, but I think that may be hard to avoid in that case.)

Reply:
I think just because there's no benefit to it in private APIs and one line is better than two lines.^^

I think it does improve code readability a bit because now you can just see in `main.rs` where exactly these methods are defined.

(btw thanks for changing this in #2801)